templates/contribs.html

```diff
@@ -6,18 +6,18 @@
         <div class="content">
             <h1>People whom have contributed to FOSSBots</h1>
             <ul>
-                <li><a href="https://meta.wikimedia.org/wiki/User:MacFan4000">MacFan4000</a> - Director of Site Reliability Engineering</li><br/>
-                <li><a href="https://meta.miraheze.org/wiki/User:RhinosF1">RhinosF1</a> - Software Engineering Lead</li><br/>
+                <li><a href="https://meta.miraheze.org/wiki/User:RyanK (FOSSBots)">Sario</a> - Director of Operations, Chair and Secretary to the Board</li><br/>
+                <li><a href="https://meta.miraheze.org/wiki/User:SamuelK (FOSSBots)">RhinosF1</a> - Software Engineering Lead</li><br/>
+                <li><a href="https://meta.wikimedia.org/wiki/User:MacFan4000">MacFan4000</a> - Site Reliability Engineering Lead</li><br/>
+                <li><a href="https://meta.miraheze.org/wiki/User:Zppix">Zppix</a> - Ethics Advisor, Project Founder and former Head of Operations</li><br/>
+                <li><a href="https://github.com/hasandiwan">Hasan Diwan</a> - Business Advisor and Code Reviewer</li><br/>
                 <li><a href="https://meta.miraheze.org/wiki/User:Void">Voidwalker</a> - Developer and Sysadmin</li><br/>
-                <li><a href="https://meta.miraheze.org/wiki/User:Zppix">Zppix</a> - Founder and former Head of Operations, current Ethics Advisor</li><br/>
-                <li><a href="https://meta.miraheze.org/wiki/User:Reception123">Reception123</a> - On-Call Sysadmin</li><br/>
-                <li><a href="https://meta.miraheze.org/wiki/User:Paladox">Paladox</a> - Assistant Developer and Advisor</li><br/>
-                <li><a href="https://github.com/hasandiwan">Hasan Diwan</a> - Advisor and Reviewer</li><br/>
-                <li><a href="https://meta.miraheze.org/wiki/User:Sario528">Sario</a> - Director of Operations, Chair and Secretary of the board </li><br/>
                 <li><a href="https://meta.miraheze.org/wiki/User:Dmehus">Doug</a> - Community Relations Specialist and PA to the Technology Department.</li><br/>
```

Comment:
Probably should be updated as well, to at least fromer.

Reply:
Yes, will do